Dockerfile

```diff
@@ -12,11 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG BASE_IMAGE=nvcr.io/nvidia/tritonserver:24.12-py3
-ARG TRITONSDK_BASE_IMAGE=nvcr.io/nvidia/tritonserver:24.12-py3-sdk
+ARG BASE_IMAGE=nvcr.io/nvidia/tritonserver:25.01-py3
```

Comment:
should this be 25.01?

Reply:
Not sure I understand, it is 25.01?

Reply:
thougth we were already at 25.02
but @mc-nv has the correct comment. This actually doesn't need to be done at all.